docs/installing-rancher/installing-server/basic-ssl-config/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Basic SSL Rancher Server Configuration
+layout: default
+---
+
+## Installing Rancher Server With SSL
+---
+
+In order to run Rancher Server from an https url, you will need to terminate SSL with a proxy that is capable of setting headers. This outlines the steps for NGINX, but you could use other tools.
+
+## Requirements
+In addition to Rancher Server requirements, you will also need:
+
+* Valid SSL certificate
+* DNS entries configured
+
+## Launching Rancher Server
+
+In this configuration, all traffic will pass through the proxy and be sent over a Docker link to the Rancher Server container. There are alternative approaches that could be followed, but this approach is simple and translates well. 
+
+Start the Rancher Server container
+
+```
+sudo docker run -d --restart=always --name=rancher-server \
+-e "CATTLE_API_ALLOW_CLIENT_OVERRIDE=true" \
+-e "CATTLE_HOST_API_PROXY_SCHEME=wss" rancher/server
+```
+
+*Note: This is assuming you will run your proxy in a container. If you are going to run a proxy from the host, you will need to expose port 8080* 
+
+If you are converting an existing environment over, stop the existing server container, and start with `--volumes-from=<old container>`
+
```

Comment:
We are now recommending people to create a data only container and then doing volumes-from the data only container. Since we use `--restart=always` and people reboot the machine, the old container will start up again (even tho it was in a stopped state.)